plat/rpi3/include/platform_def.h

```diff
@@ -64,12 +64,32 @@
  * there is no Secure RAM in the Raspberry Pi 3.
  */
 #define SEC_ROM_BASE			ULL(0x00000000)
+#if RPI3_USE_UEFI_MAP
```

Comment:
Rather than this mess of ifdefs, I think it would be better to have two sets of definitions:
```
#if RPI3_USE_UEFI_MAP
...
#else
...
#endif
```
If we have to repeat some definitions, that's ok as long as the result is more readable.

Reply:
Okay. I thought about that too, but I didn't want to duplicate the comments that come before `SEC_ROM_BASE` and `PLAT_RPI3_FIP_BASE`.

I'll modify the proposal to use a single `#if` when I get a chance.